plugins/prometheus/alerta_prometheus.py

```diff
@@ -90,8 +90,12 @@ def status_change(self, alert, status, text):
 
             # example r={"status":"success","data":{"silenceId":8}}
             try:
-                silenceId = r.json()['data']['silenceId']
-                alert.attributes['silenceId'] = silenceId
+                data = r.json().get('data', [])
+                if len(data) > 0:
```

Comment:
Can be simplified as:
```
if data:
    ...
```

Reply:
Thanks a lot, @satterly . I have updated.